### PR TITLE
APEXMALHAR-2224 Fixed GenericFileOutputOperator rotateCall.

### PR DESCRIPTION
**Pull request by prasannapramod**
@yogidevendra please review

### CODE REVIEW

**Comment by yogidevendra** [comment]
Commit message should mention ticket id. Changes looks OK.

**Comment by yogidevendra** [comment]
Could you please add ticket id in the beginning of the commit message.

**Comment by prasannapramod** [comment]
@yogidevendra i did the changes. thanks

**Comment by yogidevendra** [comment]
@prasannapramod One more small change. PR title should also be updated to start with ticket id. 
This will make sure that link for the PR will be auto populated on the JIRA ticket page. Also, JIRA will be updated after PR is merged.

**Comment by prasannapramod** [comment]
@yogidevendra I changed the PR and commit to start with the title, however it is not reflecting in the JIRA possibly because it is being changed after the original pull request was submitted. If you want me to close it and reopen a new pull request let me know.

**Comment by prasannapramod** [comment]
@yogidevendra just friendly remainder.

**Comment by yogidevendra** [comment]
http://apex.apache.org/contributing.html asks committer to ensure that pull requests will be linked to ticket. Could you please do the needful? Sorry, for the repeated back-and-forth.

**Comment by prasannapramod** [comment]
@yogidevendra can you check now. Thanks

File: library/src/test/java/org/apache/apex/malhar/lib/fs/GenericFileOutputOperatorTest.java
Comment (bhupeshchawda) [inline library/src/test/java/org/apache/apex/malhar/lib/fs/GenericFileOutputOperatorTest.java]: Can we reduce this sleep? Or even get rid of it if that makes sense for the test?
If the check is to see no data is sent in the windows, then that should work?

**Comment by prasannapramod** [comment]
@bhupeshchawda Done suggested change.

**Comment by prasannapramod** [comment]
@bhupeshchawda just friendly remainder.

**Review by bhupeshchawda — APPROVED** [review]

**Comment by bhupeshchawda** [comment]
@prasannapramod Can you please rebase the PR with latest master? Will merge after that.

**Comment by prasannapramod** [comment]
@bhupeshchawda I rebased the PR. Thanks

**Pull request merged**
